ReactWindows/ReactNative.Net46/DevSupport/WebSocketJavaScriptExecutor.cs

```diff
@@ -0,0 +1,253 @@
+﻿using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+using ReactNative.Bridge;
+using ReactNative.Collections;
+using SocketMessaging;
+using System;
+using System.Collections.Generic;
+using System.Net;
+using System.Runtime.ExceptionServices;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace ReactNative.DevSupport
+{
+    class WebSocketJavaScriptExecutor : IJavaScriptExecutor
```

Comment:
Any tests, even minor ones, that could be added for this? What was the manual test case?